drivers/sensors/Kconfig

```diff
@@ -838,6 +838,35 @@ config KXTJ9_I2C_BUS_SPEED
 
 endif # SENSORS_KXTJ9
 
+config SENSORS_L86_XXX
+	bool "Quectel L86-XXX GNSS support"
+	default n
+	depends on SERIAL && STANDARD_SERIAL && UORB
+	---help---
+		Enable driver support for the L86-XXX series of GNSS modules.
```

Comment:
@EliasJRH please put an information here explaining that this driver needs to me improved and use the uORB GNSS upperhalf as @xiaoxiang781216 suggested